dts/riscv/riscv32-litex-vexriscv.dtsi

```diff
@@ -61,7 +61,7 @@
 			#address-cells = <1>;
 			#size-cells = <0>;
 		};
-		timer0: serial@e0002800 {
+		timer0: timer@e0002800 {
```

Comment:
nit: this is an unrelated change so please split out into a different PR.

Reply:
You are right, this is not related to I2S - that's why it's put into a separate commit. Is there a point of creating a separate PR just for this small change? If so, we'll ofc split this.

Reply:
I extracted this commit to a separate PR: #27519